docs/conf.py

```diff
@@ -352,3 +421,10 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
+
+
+linkcheck_ignore = [
+    r"https://discord.com/developers/docs/.*#",
```

Comment:
is .*# correct?

Reply:
> is .*# correct?

Yes, see [this note on ignoring anchors](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_anchors_ignore) in the Sphinx documentation.